delivery_carrier_label_batch/models/stock_batch_picking.py

```diff
@@ -0,0 +1,129 @@
+# Copyright 2013-2019 Camptocamp SA
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl)
+from odoo import Command, api, fields, models
+
+
+class StockBatchPicking(models.Model):
+    """Add carrier and carrier options on batch
+
+    to be able to massively set those options on related picking.
+
+    """
+
+    _inherit = "stock.picking.batch"
+
+    carrier_id = fields.Many2one(
+        comodel_name="delivery.carrier",
+        string="Carrier",
```

Comment:
Redundant `string`

Reply:
why redundant? as I know many2x fields aren't handled automatically. Am I wrong?

Reply:
Odoo defaults to "Carrier" by formatting the technical name (`carrier_id`): https://github.com/odoo/odoo/blob/18.0/odoo/fields.py#L503-L508